mofacts/client/views/experiment/unitEngine.js

```diff
@@ -1039,7 +1037,14 @@ function modelUnitEngine() {
           });
         }
       }
-      Session.set('numVisableCards', numVisableCards - hiddenItems.length);
+
+      let numVisibleCards = 0;
+      for (let i = 0; i < cardProbabilities.cards.length; i++){
+        if(cardProbabilities.cards[i].canUse){
```

Comment:
i thought you were using the kc num to identify hidden/visible cards?

Reply:
This is used to find the total number of cards that were visable to the user through the <clusterlist> tag in the tdf file. If a cluster is being used by the tdf then the canUse parameter is set to true. This is just to get the total number of stims enabled in the tdf. Afterwards the number of hidden items is subtracted from the total.

Reply:
Ah, carry on then